docs/content/en/tasks/monitor/vector/_index.md

```diff
@@ -107,7 +143,33 @@ The above configuration should display the scraped Cassandra metrics in json for
 {"name":"org_apache_cassandra_metrics_table_bloom_filter_off_heap_memory_used","tags":{"cluster":"Weird Cluster Name","datacenter":"dc1","host":"95c50ce3-2c91-46bb-9500-536f0959241b","instance":"10.244.2.8","keyspace":"system_auth","rack":"default","table":"network_permissions"},"timestamp":"2023-02-02T10:28:08.670070700Z","kind":"absolute","gauge":{"value":0.0}}
 ```
 
-Providing no custom configuration would produce the following configmap:
+As an another example, to output the logs as json to the console, one could modify the configuration to following:
+
+```yaml
+cassandra:  
+  telemetry:
+      vector:
+        enabled: true
+        components:
+          sinks:
+            - name: console_output
+              type: console
+              inputs:
+                - parse_cassandra_log
+              config: |
+                target = "stdout"
+                [sinks.console_output.encoding]
+                codec = "json"
+        scrapeInterval: 30s
+        resources:
+          requests:
+            cpu: 1000m
+            memory: 1Gi
+          limits:
+            memory: 2Gi
```

Comment:
These aren't taken into account, right?
Do we have a ticket to track making these values correctly mapped to the cassdc object (and what that mapping should be)?

Reply:
Well, they're read - but they apply them to the wrong place in the CassDc (they do it like the tests say.. but tests are wrong ;)). 

There's another PR, but no ticket.

Reply:
No, there was a ticket.. https://github.com/k8ssandra/k8ssandra-operator/issues/940